lib/client.js

```diff
@@ -1,29 +1,22 @@
 (function (global, factory, undefined) {
 
-  if ( typeof module === 'object' && typeof module.exports === 'object' ) {
-    var ActivityStreams = require('activity-streams');
+  if ( typeof module === "object" && typeof module.exports === "object" ) {
```

Comment:
Namedropping isn't exactly an argument for either side. I think if anything, it should be consistent, and if we want to change it to double quotes (which I personally don't see a reason for, but I couldn't care less), then we should do it across the board and introduce linting for it, instead of just randomly starting to mix in a different style.